trajectory_msgs/JointTrajectoryPoint jointTrajectoryPoint
float64[] epsTau
float64 dtList
float64 motorOn
